packages/experimental/Dropdown/src/Option/Option.styling.ts

```diff
@@ -3,4 +3,27 @@ import { optionName, OptionProps, OptionSlotProps, OptionTokens } from './Option
 
 export const stylingSettings: UseStylingOptions<OptionProps, OptionSlotProps, OptionTokens> = {
   tokens: [optionName],
+  slotProps: {
+    root: {
+      style: {
+        alignItems: 'center',
+        alignSelf: 'flex-start',
+        borderColor: 'black',
+        borderRadius: 2,
+        borderWidth: 1,
```

Comment:
I saw you wrote that token handling will be in future PR. Will borderStyles, also, be used in the following PR?

Reply:
Yeah, I'll update the border styles in a future PR, this was just to make the Option presentable